cli.rs

```diff
@@ -120,13 +120,22 @@ fn run() -> Result<(), Error> {
                 .iter()
                 .filter(|x| filter.as_ref().map(|f| x.contains(f)).unwrap_or(true));
 
-            if infinite {
-                for target in targets.cycle() {
-                    run(target)?;
+            'until_the_end_of_time: loop {
+                'all_the_fuzz: for target in targets.clone() {
```

Comment:
I would suggest other labels like
`'infinite_cycle` and `'targets_pass`.

Reply:
How boring… but ooookay